apps/accounts/tests/test_verify.py

```diff
@@ -0,0 +1,105 @@
+# No django imports.
+
+import mock
+
+from apps.accounts import verify
+
+
+class TestProfile(object):
+
+    @mock.patch('apps.accounts.verify.requests.get')
+    def test_get_profile_success(self, get):
+        profile_data = {'email': 'yo@oy.com'}
+        get.return_value.status_code = 200
+        get.return_value.json.return_value = profile_data
+        profile = verify.get_fxa_profile('profile-plz', {
+            'profile_uri': 'https://fxa.com/v1',
+        })
+        assert profile == profile_data
+        get.assert_called_with('https://fxa.com/v1/profile', headers={
+            'Authorization': 'Bearer profile-plz',
+        })
+
+    @mock.patch('apps.accounts.verify.requests.get')
```

Comment:
when there are more than one patch like this, I find putting them in setUp is a little more maintainable. I know it adds more code but it feels cleaner.